sdk/core/azure-core/azure/core/pipeline/transport/_base.py

```diff
@@ -222,16 +222,20 @@ class HttpRequest(object):
     :param files: Files list.
     :param data: Body to be sent.
     :type data: bytes or str.
+    :keyword dict[str,any] query: A dictionary of query parameters you would like to include
```

Comment:
The `query` parameter should take a sequence of key-value tuples in addition to a `dict`. Many services allow/require the same query parameter multiple times (something similar to `http://contoso.com/api?item=1&item=2&item=7`)

Reply:
@johanste would it be ok to keep it as `Dict[str, Any]` for this release so we get the `query` kwarg in place? Currently core only accepts `Dict[str, Any]` in `format_parameters`, so this would be at parity. For next release, I can open an issue to expand query to allow in lists of tuples etc, following [httpx's possible values](https://github.com/encode/httpx/blob/master/httpx/_types.py#L34)